### DISCUSSION TITLE
article/how-to-develop-with-notion-next

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#54] [Announcements]
# article/how-to-develop-with-notion-next

小白入门如何开发notionnext，下载NotionNext、启动NotionNext、提交合并到NotionNext等。

https://tangly1024.com/article/how-to-develop-with-notion-next

**Comment by Muzych** [comment]
你好博主，我在使用fukasawa主题的时候，感觉到侧边栏如果能加上一个expand-collapse的控件会提高使用体验，请问在fukasawa中有没有相关的配置呀？
